src/WSRequestHandler_General.cpp

```diff
@@ -304,3 +304,32 @@ RpcResponse WSRequestHandler::GetVideoInfo(const RpcRequest& request) {
 
 	return request.success(response);
 }
+
+/**
+ * Open a projector window or create a projector on a monitor. Requires OBS v24.0.4 or newer.
+ * 
+ * @param {String (Optional)} `type` Type of projector: Preview (default), Source, Scene, StudioProgram, or Multiview (case insensitive).
+ * @param {int (Optional)} `monitor` Monitor to open the projector on. If -1 or omitted, opens a window.
+ * @param {String (Optional)} `geometry` Size and position of the projector window (only if monitor is -1). Encoded in base 64.
```

Comment:
Could you please add a bit more info on the format of the data before it gets encoded in Base64, please? So that users get to know how to generate the geometry string.

Reply:
It seems that the geometry string is based entirely on Qt.

https://doc.qt.io/qt-5/qwidget.html#saveGeometry

I'm inclined to not try to document the actual format, but just to report that this ties in with the saved projectors in OBS's own configs. Maybe best would be to have a way to query all current projectors?

Reply:
Alright then. I'll add a mention to Qt's documentation and merge this.